src/main/java/com/powsybl/balances_adjustment/data_exchange/DataExchanges.java

```diff
@@ -81,6 +87,8 @@ data base system (e. g. Verification Platform). */
         this.datasetMarketDocumentMRId = datasetMarketDocumentMRId;
         this.docStatus = docStatus;
         this.timeSeriesById.putAll(timeSeriesById);
+        this.domainId = domainId;
+        this.domainCodingScheme = domainCodingScheme;
```

Comment:
Maybe just check that if `domainId` not null, `domainCodingScheme` should also not be null?